app/models/pin.rb

```diff
@@ -34,17 +45,15 @@ class Pin < ActiveRecord::Base
     end
   end
 
-  belongs_to :user
-  belongs_to :surgeon
-  belongs_to :procedure
-
-  has_many :pin_images, dependent: :destroy
-
-  attr_accessor :pin_image_ids
-
-  acts_as_commentable
-  acts_as_votable
-  acts_as_taggable_on :complications
+  def as_indexed_json(options={})
```

Comment:
note: still want to clean this up as it is not really the right format, but it works for now.